packages/terra-date-time-picker/src/terra-dev-site/test/date-time-picker/DateTimePickerWithTimezoneMissingHour.test.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import DateTimePickerTimezoneExampleTemplate from '../../common/DateTimePickerTimezoneExampleTemplate';
+
+const DateTimePickerExample = () => (
+  <div>
```

Comment:
the div wrapper prob isn't needed

Reply:
Removed here b5f07bc5f04b127f3d75bc46a3651a2e1091b165